Add holiday roles

TheHans255 opened the pull request:
# Description

- Add Bloody Holly, Elf and Safety Ossifer, Santa Claws, and Rude Dolph Lundgren.
- Add them under the category "Holiday Roles" to keep them separate from the standard roles.
- Add descriptions, role types, notes, and history for each, leaving Tips blank

# Checklist

I have checked the following _(mark as completed with [x])_:

- [x] The content is accurate (and confirmed with the mods)
- [x] Casing and wording of keywords such as role names, factions, etc, is all correct
- [x] The Markdown is valid
- [x] It renders correctly _(follow README for instructions on previewing)_
- [x] The sidebar links are correct and in the right sections _(follow README for instructions on previewing)_

Commit:
Add holiday roles

- Add Bloody Holly, Elf and Safety Ossifer, Santa Claws, and
  Rude Dolph Lundgren.
- Add them under the category "Holiday Roles" to keep them separate
  from the standard roles.
- Add descriptions, role types, notes, and history for each, leaving
  Tips blank

Kirschstein commented on src/guides/zaa_holiday-roles/santa-claws.mdx:
```diff
@@ -0,0 +1,33 @@
+
+## Notes
+
+- Santa Claws is aligned to the werewolves (TODO, do they know who they are?), but is not part of their night chat.
```
neither party knows who the other is

TheHans255 replied:
OK, will add

Commit:
Merge branch 'site' into thehans255/holiday-roles

Commit:
Confirm that Santa Claws doesn't know the wolves

Commit:
Add the Holiday Roles page to doczrc.js

Commit:
Small changes and censors to Bloody Holly

- Mention that a blood soaked wreath will cause
the holder to appear as that faction
- Do not mention what specific blood types
the wreaths can be soaked with

Kirschstein committed:
Merge branch 'site' into thehans255/holiday-roles
